gtsfm/loader/astronet_loader.py

```diff
@@ -0,0 +1,242 @@
+"""Simple loader class that reads a dataset with metadata formatted in the COLMAP style.
+
+Authors: Travis Driver
+"""
+
+import os
+from pathlib import Path
+from typing import Optional, Dict, Tuple, List
+
+from gtsam import Cal3Bundler, Pose3, Rot3, Point3, SfmTrack
+import trimesh
+
+import gtsfm.utils.io as io_utils
+import gtsfm.utils.logger as logger_utils
+from gtsfm.common.image import Image
+from gtsfm.loader.loader_base import LoaderBase
+
+from gtsfm.utils.read_write_model import read_model
```

Comment:
thanks for the PR, Travis. Very cool.

Usually we only import classes from modules, otherwise we just import the module by its name, e.g. `import gtsfm.utils.read_write_model as read_write_model`

Maybe a better name for that module is `mesh_utils` or `model_utils` or `model_io` or `tri_model_io`?

Reply:
Oh nevermind, I see that is the original COLMAP name for it? if so, we should put it in the `third_party` folder. What functionality do we need from it that we don't already have in `gtsfm.utils.io`?

Reply:
`gtsfm.utils.io` doesn't have any of the binary read/write. I can add it thought if you want.

Reply:
which data is stored in binary format?

Reply:
Everything except for the images

Reply:
Moved this file to `thirdparty.colmap.scripts.python.read_wrtie_model`.